tests/test_collections.py

```diff
@@ -17,11 +19,22 @@
 from dss.util import UrlBuilder
 from tests.infra.server import ThreadedLocalServer
 
-class TestCollections(unittest.TestCase, DSSAssertMixin, DSSUploadMixin):
-    @classmethod
-    def setUpClass(cls):
-        cls.app = ThreadedLocalServer()
-        cls.app.start()
+
+app = ThreadedLocalServer()
+
+
+def setUpModule():
+    global app
+    app.start()
+
+
+def tearDownModule():
+    global app
+    app.shutdown()
+
+
+class CollectionBase(DSSAssertMixin, DSSUploadMixin):
```

Comment:
What's the reason for extracting this base class?

Reply:
I was going to have to have multiple unittest.TestCases that subclasses this so I separated out all of the common components. I'll recombine everything.